scripts/build-package.js

```diff
@@ -4,26 +4,21 @@
 const { resolve, join, posix, sep } = require('path');
 const { readJSON } = require('fs-extra');
 
-const getStorybookPackages = async () => {
-  const process = require('util').promisify(require('child_process').exec);
-  const contents = await process('lerna ls --json', { cwd: join(__dirname, '..', 'code') });
-
-  const projects = JSON.parse(contents.stdout);
-  return projects.reduce((acc, project) => {
-    acc.push({
-      name: project.name,
-      location: project.location,
-    });
-    return acc;
-  }, []);
-};
+async function getWorkspaces(includePrivate = true) {
+  const { execaCommand } = await import('execa');
+  const { stdout } = await execaCommand(`yarn workspaces list --json --no-private`, {
+    cwd: join(__dirname, '..', 'code'),
+    shell: true,
+  });
+  return JSON.parse(`[${stdout.split('\n').join(',')}]`);
+}
```

Comment:
Almost the same as the function in `scripts/utils/workspace.ts`. But this is a Javascript file, from where I cannot import Typescript files I guess.

Reply:
convert it to TS?

Reply:
node doesn't execute ts, does it? So I have to use ts-node instead to run the script?